### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by BirgitBader**

### CODE REVIEW

File: markdown_stoplight/changelog.md [hunk 1]
```diff
@@ -3,118 +3,149 @@
...
 endpoint.
 
 **Update**
 
 Change [Enable support access](#resources-enable-support-access)
 endpoint by removing user details from the response.
 
-Change [List users](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/users/enable-support-access) and [Show user
-details](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/users/user-get) endpoints by removing support user
+Change [List users](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/users/enable-support-access) and [Show user
```
Comment (cgebken) [inline markdown_stoplight/changelog.md:130]: Weird line break

File: markdown_stoplight/changelog.md [hunk 2]
```diff
@@ -126,14 +157,14 @@ Enhance official apps to carry additional configuration data used for
 
 **New**
 
-Add [Enable support access](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/users/enable-support-access) and
-[Disable support access](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/users/enable-support-access) user
+Add [Enable support access](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/users/enable-support-access) and
+[Disable support access](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/users/enable-support-access) user
```
Comment (cgebken) [inline markdown_stoplight/changelog.md]: Weird line break

File: markdown_stoplight/changelog.md [hunk 3]
```diff
@@ -3,118 +3,149 @@
...
 **Update**
 
-Change [cart details](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/carts/cart-create) by adding
-*mustAcceptTermsAndConditions* property in the payload.
+Change [cart details](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/carts/cart-create) by adding `mustAcceptTermsAndConditions` property in the payload.
 
-Change [Create order from cart](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/carts/cart-order) endpoint by
-adding *termsAndConditionsExplicitlyAccepted* property in the payload.
+Change [Create order from cart](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/carts/cart-order) endpoint by
+adding `termsAndConditionsExplicitlyAccepted` property in the payload.
```
Comment (cgebken) [inline markdown_stoplight/changelog.md]: Weird line break

File: markdown_stoplight/changelog.md [hunk 4]
```diff
@@ -3,118 +3,149 @@
...
 **New**
 
-Change [shop resource's body](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/shop/merchant-shop-get) by
+Change [shop resource's body](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/shop/merchant-shop-get) by
 re-working its structure and content.
 
 Add [shop attributes
-resource](https://beyond-rest-api.docs.stoplight.io/beyond-rest-api/api-reference/shop/merchant-shop-attribute-create), which allow
-clients to store arbitrary information.
+resource](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/shop/merchant-shop-attribute-create), which allow clients to store arbitrary information.
```
Comment (cgebken) [inline markdown_stoplight/changelog.md]: Weird line break

Which allowS ?

File: markdown_stoplight/changelog.md [hunk 5]
```diff
@@ -3,118 +3,149 @@
 This change log covers significant updates and changes to the Beyond
 REST API.
```
Comment (cgebken) [inline markdown_stoplight/changelog.md]: Weird line break

File: markdown_stoplight/changelog.md [hunk 6]
```diff
@@ -3,118 +3,149 @@
...
+
+Add field `captureWorkflow` to payment method definitions.
+
+**2018-12-18**
+
+**Update**
+
+Add optional field `dependentLocality` to billing and shipping address properties.
+
+Add payment status for prepayment-style payments (e.g. OVERPAID, UNDERPAID). Allow passing the actual amount paid to the [Set payment status with amount](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/payments/set-payment-status-with-amount) endpoint (first implementation step).
```
Comment (cgebken) [inline markdown_stoplight/changelog.md:23]: Tough to read for a change log. Maybe split it into two entries?

File: markdown_stoplight/changelog.md [hunk 7]
```diff
@@ -1,120 +1,147 @@
...
-REST API.
+This change log covers significant updates and changes to the Beyond REST API.
+
+**2019-01-15**
+
+**New**
+
+Allow user creation without specifying a password.
+
+Add new endpoint [Create variation product](https://beyond.docs.stoplight.io/beyond-rest-api/api-reference/products/products-create-with-variations).
```
Comment (jensfischer1515) [inline markdown_stoplight/changelog.md:11]: I wonder if we already want to document this, while not being able to ship this feature to all merchants, yet?

Comment (BirgitBader) [reply markdown_stoplight/changelog.md:11]: I talked to Jan and Timo, and it's fine for them.

**Pull request merged**